cmd/openshift-tests/e2e.go

```diff
@@ -401,13 +401,13 @@ func suiteWithInitializedProviderPreSuite(opt *runOptions) error {
 }
 
 // suiteWithProviderPreSuite ensures that the suite filters out tests from providers
-// that aren't relevant (see getProviderMatchFn) by loading the provider info from the
-// cluster or flags.
+// that aren't relevant (see exutilcluster.ClusterConfig.MatchFn) by loading the
```

Comment:
This refactor is fine, you could also pull this out into its own PR to reduce churn in this one